### PR TITLE
fixes for daniel stuff

### PR DESCRIPTION
**Pull request by everyonce**

### CODE REVIEW

**Comment by lydonb** [comment]
:shipit:

File: scripts/daniel.coffee [hunk 1]
```diff
@@ -14,9 +14,11 @@ module.exports = (robot) ->
...
             .get() (error, response, body) ->
-                data = JSON.parse body
-                msg.send data.thought
+                data = JSON.parse(body)
+                msg.send decodeURI(data.thought.replace(/&quot;/g,'"'))
 
     robot.respond /room/i, (msg) ->
-        msg.send robot.messageRoom()
+        room = msg.message.room
+        console.log('The room is ' + room)
```
Comment (newtonrd) [inline scripts/daniel.coffee]: Do you want to keep the console.log in?

**Comment by everyonce** [comment]
I fixed/removed the console log

**Comment by lydonb** [comment]
Squash those commits homie.

**Comment by everyonce** [comment]
ok, squashed

**Comment by lydonb** [comment]
Donezo

**Pull request merged**